decide/authentication/test-selenium.py

```diff
@@ -51,3 +51,83 @@ def test_failed_login(self):
+    def setUp(self):
+        self.base = BaseTestCase()
+        self.base.setUp()
+
+        # Opciones de Chrome
+        options = webdriver.ChromeOptions()
+        options.headless = True
+        options.add_argument("--no-sandbox")
+        self.driver = webdriver.Chrome(options=options)
+        self.user = User.objects.create_user(username="testuser", password="testpass")
+        super().setUp()
+
+    def tearDown(self):
+        super().tearDown()
+        self.driver.quit()
+
+        self.base.tearDown()
+
+    def test_sucessful_registration(self):
+        self.driver.get(f"{self.live_server_url}/register")
+
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_username")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_first_name")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_last_name")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_email")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_password1")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_password2")) == 1)
+
+        self.driver.find_element(By.ID, "id_username").send_keys("new_user")
+        self.driver.find_element(By.ID, "id_first_name").send_keys("John")
+        self.driver.find_element(By.ID, "id_last_name").send_keys("Doe")
+        self.driver.find_element(By.ID, "id_email").send_keys("john.doe@example.com")
+        self.driver.find_element(By.ID, "id_password1").send_keys("strong_password123")
+        self.driver.find_element(By.ID, "id_password2").send_keys("strong_password123")
+        self.driver.find_element(By.CSS_SELECTOR, ".btn").click()
+
+        self.assertTrue(self.driver.title == "Decide | Login")
+
+    def test_failed_registration(self):
+        self.driver.get(f"{self.live_server_url}/register")
+
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_username")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_first_name")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_last_name")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_email")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_password1")) == 1)
+        self.assertTrue(len(self.driver.find_elements(By.ID, "id_password2")) == 1)
+
+        self.driver.find_element(By.ID, "id_username").send_keys("")
+        self.driver.find_element(By.ID, "id_first_name").send_keys("")
+        self.driver.find_element(By.ID, "id_last_name").send_keys("")
+        self.driver.find_element(By.ID, "id_email").send_keys("john")
+        self.driver.find_element(By.ID, "id_password1").send_keys("strong_password")
+        self.driver.find_element(By.ID, "id_password2").send_keys("strong_password123")
+        self.driver.find_element(By.CSS_SELECTOR, ".btn").click()
+
+        self.assertTrue(self.driver.title == "Decide | Registration")
```

Comment:
This test does not send a request and get an error back, it simply rejects the form contents before sending it to the backend. Please add another test with, for example, a password that is too short to check that everything works as intended